python/lsst/meas/extensions/ngmix/LMSimpleShape.py

```diff
@@ -206,19 +207,23 @@ def measure(self, measRecord, exposure):
                     SingleFrameLMSimpleShapePlugin.ErrEnum.flag_singularMatrix)
             if res['flags'] & LM_NEG_COV_EIG:
                 raise MeasurementError(
-                    self.flagHandler.getDefinition(SingleFrameLMSimpleShapePlugin.ErrEnum.flag_negativeCovEig).doc,
+                    self.flagHandler.getDefinition(
+                        SingleFrameLMSimpleShapePlugin.ErrEnum.flag_negativeCovEig).doc,
```

Comment:
This is super weird because it's argument 1 but argument 2 on the next line never moved. Does `flake8` really not complain for this?